katalog/cert-manager/README.md

```diff
@@ -11,17 +11,17 @@ Encrypt](https://letsencrypt.org/) Certificate Authority.
 
 -   Kubernetes >= `1.10.0`
 -   Kustomize >= `v1`
--   [nginx-ingress]()
+-   [nginx-ingress](../nginx)
 
 ## Image repository and tag
 
--   Cert Manager image: `quay.io/jetstack/cert-manager-controller:v0.9.0`
+-   Cert Manager image: `quay.io/jetstack/cert-manager-controller:v0.14.1`
 -   Cert Manager repo: https://github.com/jetstack/cert-manager
--   Cert Manager documentation: https://docs.cert-manager.io/en/release-0.9/index.html
+-   Cert Manager documentation: https://cert-manager.io/docs/
```

Comment:
This change does give me some thoughts about certs of kubernetesfury.com (not related to this PR)